src/assignments/ray/lesson_2_answers.clj

```diff
@@ -1,3 +1,34 @@
 (ns lesson-2-answers)
 
 ;;; Enter your answers in here.
+(def words_1 ["meat" "mat" "team" "mate" "eat"])
+(first words_1)
+(rest words_1)
+(def words_2 ["veer" "lake" "item" "kale" "mite" "ever"])
+
+(defn toS [words]
```

Comment:
We also tend to write legible, full words, rather than abbreviations
(altough not in the arguments of functions, often).  I know the stylistic
matters will feel foreign to you for a while, but you'll get used to it as
you read a lot of existing code.

so, to-set is a better identifier than toS.

Also, we very rarely use capital letters; we tend to automatically associate
those words with Java interop functions (as Java of course uses CamelCase with
caps letters all over the place.